alchemy/src/cloudflare/wrangler.json.ts

```diff
@@ -508,6 +508,8 @@ function processBindings(
       });
     } else if (binding.type === "json") {
       // TODO(sam): anything to do here? not sure wrangler.json supports this
+    } else if (binding.type === "dispatch_namespace") {
+      // TODO(sam): dispatch namespaces are not supported in wrangler.json yet
```

Comment:
I don't think this is true, see https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/get-started/configuration/

{
  "dispatch_namespaces": [
    {
      "binding": "DISPATCHER",
      "namespace": "testing"
    }
  ]
}